test/services/FootprintService.test.js

```diff
@@ -149,14 +149,8 @@ describe('api.services.FootprintService', () => {
         .then(role => {
           assert(role)
           assert(role.id)
-          return FootprintService.find('User', userId)
+          assert.equal(role.dataValues.UserId, userId)
         })
-        .then(user => {
-          assert(user[0])
-          assert.equal(user[0].roles.length, 1)
-          assert.equal(user[0].roles[0].name, 'createassociatedrole')
-        })
```

Comment:
Should this work? `user.roles` was never defined when I tested it, and it should also be user not user[0]